src/components/IFrame.js

```diff
@@ -42,23 +43,25 @@ export default class IFrame extends React.Component<Props, State> {
 
     this.origin = 'https://commerce.coinbase.com';
     this.uuid = generateUUID();
-    this.hostName = `${window.location.protocol}//${window.location.hostname}${window.location.port ? ':' + window.location.port : ''}`;
 
     this.state = {
-      loading: true
+      loading: true,
+      src: null
     }
   }
 
   componentDidMount(){
     // Add event listeners for the iframe
     window.addEventListener('message', this.handleMessage);
+    const hostName = `${window.location.protocol}//${window.location.hostname}${window.location.port ? ':' + window.location.port : ''}`;
```

Comment:
nit: you could use [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) to normalize this